Tiltfile

```diff
@@ -693,7 +693,7 @@ if aptos:
         dockerfile = "aptos/Dockerfile",
         only = ["Dockerfile", "node_builder.sh", "start_node.sh", "README.md", "cert.pem"],
     )
-
+    
```

Comment:
You're adding trailing whitespace. Please fix your editor. I like to set it to strip trailing whitespace on buffer save.